plugins/commands.js

```diff
@@ -21,6 +23,15 @@ exports.plugin = {
      * @param  {Integer}  options.maxByteSize   Maximum Bytes to accept
      */
     register(server, options) {
+        const segment = 'commands';
```

Comment:
Maybe set to constant?

Reply:
@tkyi do you mean to update const segment = 'commands'

Reply:
sorry I meant like this: 
```suggestion
        const SEGMENT = 'commands';
```
maybe move as global var

Reply:
kept it same as https://github.com/screwdriver-cd/store/blob/44eb3492c83602ddf0bcad034482ab9b46b5f7a9/plugins/builds.js#L34